src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx

```diff
@@ -61,7 +51,19 @@ For a detailed comparison of capabilities, see [the capabilities table](/docs/ac
 This pricing version also changes [how billable users are calculated](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#user-calculation) and [rules on downgrading users](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#downgrade-rules).
 
 
-## Details on opting in to this pricing version [#nr1core-optin]
+## Availability [#availability]
+
+The core users release is currently available to: 
+* New Relic organizations that were created on or after January 12, 2022. 
+* Organizations that existed before that date that meet certain requirements. If it's available to your organization, you'll receive messaging about that, either via [opt-in messaging](#nr1core-optin) in your billing management UI, or from your New Relic account representative.
+
+If your organization has core users or has the ability to add core users, you're on this version. If you don't, you're on the ["New Relic One without core users" version](#overview). For tips on determining which version you're on, see [Pricing versions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#pricing-versions).
+
+### Free access to core user capabilities [#free-access] 
+
+For all New Relic organizations, their basic users will have free preview access to core user capabilities until February 28, 2022. Core user capabilities are described below. 
+
+## Details on opting in [#nr1core-optin]
```

Comment:
Instead of `Details on opting in` what about a nudgier `How to opt in`?

Reply:
sure, although it's only available to some orgs